src/extractors/custom/medium.com/index.test.js

```diff
@@ -78,12 +78,9 @@ describe('MediumExtractor', () => {
         13
       );
 
-      // testing that youtube video transform is working
-      assert.equal(/IAoy3ia2ivI/.test(content), true);
-
       assert.equal(
         first13,
-        'Video of WTF? My talk at the White House Frontiers ConferenceLast Thursday, I'
+        'Video of WTF? My talk at the White House Frontiers ConferenceOct 19, 2016'
```

Comment:
Is this the dek, or is it the start of the content? It looks like the dek and then a timestamp, but maybe I don't have the context right.

Reply:
Great catch. This was actually a caption for a youtube video that snuck in there. I added a fix to not draw captions for youtube vids if we weren't able to pull in the video.